Code/Engine/RendererCore/Textures/TextureUtils.h

```diff
@@ -14,6 +14,13 @@ struct EZ_RENDERERCORE_DLL ezTextureUtils
 
   static void ConfigureSampler(ezTextureFilterSetting::Enum filter, ezGALSamplerStateCreationDescription& out_sampler);
 
+  /// \brief Copies the given texture subresource from `memory` into `out_Image` according to the texture description.
+  static void CreateSubResourceImage(const ezGALTextureCreationDescription& desc, const ezGALTextureSubresource& subResource, ezGALSystemMemoryDescription& memory, ezImage& out_Image, bool bRemoveSRGB);
```

Comment:
Maybe the function should be named something like CopySubResourceToImage? Create... sounds misleading.